Composer/ScriptHandler.php

```diff
@@ -285,33 +281,15 @@ public static function doBuildBootstrap($bootstrapDir, $autoloadDir = null, $use
 
         ClassCollectionLoader::load($classes, dirname($file), basename($file, '.php.cache'), false, false, '.php.cache');
 
-        $fs = new Filesystem();
         $bootstrapContent = substr(file_get_contents($file), 5);
 
-        if ($useNewDirectoryStructure) {
-            $cacheDir = $fs->makePathRelative($bootstrapDir, $autoloadDir);
-            $bootstrapContent = str_replace(array("return \$this->rootDir.'/logs", "return \$this->rootDir.'/cache"), array("return \$this->rootDir.'/".$cacheDir.'logs', "return \$this->rootDir.'/".$cacheDir.'cache'), $bootstrapContent);
```

Comment:
should be obsolete since we do overwrite those methods in our kernel now: https://github.com/symfony/symfony-standard/commit/693e1f79e4476ab3810e786300026e63e92252bd#diff-c23da96dc8986a4cee89980f3aad30e0R32